cypress_test/integration_tests/desktop/calc/statusbar_spec.js

```diff
@@ -34,11 +34,11 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Statubar tests.', function
 	});
 
 	it('Text editing mode.', function() {
-		cy.cGet('#InsertMode').should('have.text', 'Insert mode: inactive');
+		cy.cGet('#InsertMode').should('have.text', 'Tracking changes: on');
```

Comment:
I mean there is inconsistency here. `InsertMode` shows `Tracking changes: on` - it looks like naming doesn't follow the content for me. Confusing.

Is that about "insert mode" or "changes tracking" feature? then both name of field and it's text should be consistent.

Reply:
Oh, I understand now, what you trying to mean here. I think in that case we should change the wording from "Tracking changes: on".

cc @pedropintosilva

Reply:
but I see that you were just following my other solution https://github.com/CollaboraOnline/online/issues/11244 (updated)

Reply:
Updated changes to hide "insert mode: inactive"